DFC.Api.JobProfiles/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace DFC.Api.JobProfiles.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string PrefixWithBaseAddress(this string url, HttpRequest request)
```

Comment:
Might read better swapping this around.
```suggestion
        public static string GetAbsoluteUrlForRelativePath(this HttpRequest request, string relativePath)
```

Reply:
Swapped as suggested